src/data/reference-data/references.ts

```diff
@@ -1,12 +1,64 @@
 export default {
+  T2THuman: {
+    ref_fasta: 'gs://gcp-public-data--broad-references/t2t/v2/chm13v2.0.maskedY.fasta',
+    ref_fasta_index: 'gs://gcp-public-data--broad-references/t2t/v2/chm13v2.0.maskedY.fasta.fai',
+    ref_dict: 'gs://gcp-public-data--broad-references/t2t/v2/chm13v2.0.maskedY.dict',
+    ref_amb: 'gs://gcp-public-data--broad-references/t2t/v2/chm13v2.0.maskedY.fasta.amb',
+    ref_ann: 'gs://gcp-public-data--broad-references/t2t/v2/chm13v2.0.maskedY.fasta.ann',
+    ref_bwt: 'gs://gcp-public-data--broad-references/t2t/v2/chm13v2.0.maskedY.fasta.bwt',
+    ref_pac: 'gs://gcp-public-data--broad-references/t2t/v2/chm13v2.0.maskedY.fasta.pac',
+    ref_sa: 'gs://gcp-public-data--broad-references/t2t/v2/chm13v2.0.maskedY.fasta.sa',
+    par_bed: 'gs://gcp-public-data--broad-references/t2t/v2/chm13v2.0_PAR.bed'
+  },
+  hg38Human: {
```

Comment:
Key names in this object can never be changed. This will "orphan" reference data that has been added to existing workspaces.

Reply:
Ah I was worried about that. Thanks for letting me know; I just reverted it!

Reply:
In that case, would it make sense to use a more specific key than `T2THuman`, like `T2Tv2.0_Human`?

Reply:
Possibly. In that case, we'd probably also want to decouple the keys in references.ts from the labels shown to users. That would also allow us to show hg38 as "hg38 Human".

Reply:
Do you think that would be a heavy lift? If you had a few files to suggest that might be relevant, I could try taking a look at attempting to refactor that a little (though my webdev is very beginner-level).

Reply:
Not terrible, though unfortunately there are no tests for this part of the app. If you decide to take it on, I would suggest doing it in a separate PR and rebasing this one on top of it.

The relevant files are src/data/reference-data/* and src/data/Data.js.

Reply:
Just submitted this PR [here](https://github.com/DataBiosphere/terra-ui/pull/4222) implementing your suggestion. Once that gets sorted out, I can rebase this one with the few lines needed to add the T2T reference.

Thanks again for your help!